### DISCUSSION TITLE
Are there plans to make the Analyzers support the isolated worker mode?

### DISCUSSION THREAD

**Discussion by nulltoken** [Azure/azure-functions-durable-extension#2362] [Q&A]
:wave: Hello, from what I see in the [tests](https://github.com/Azure/azure-functions-durable-extension/blob/dev/test/WebJobs.Extensions.DurableTask.Analyzers.Test/Orchestrator/DateTimeAnalyzerTests.cs#L32-L49), it looks like the Analyzers are rather targeting inProcess mode.

Are those supposed to be upgraded to also support running in a isolated worker context?

**Comment by davidmrdavid** [comment]
Pinging @jviau and @amdeel as the SMEs in these areas

**Reply by jviau** [reply]
This is on my radar, but I am unsure what it would be done. I'd like to take this opportunity to make analyzers for both Durable standalone and functions at once.
